.github/workflows/linux.yml

```diff
@@ -25,31 +25,26 @@ jobs:
     name: Try to download prebuilt OpenVINO
     outputs:
       status: ${{ steps.openvino_download.outcome }}
+      ov_package_path: ${{ steps.openvino_download.outputs.ov_package_path }}
+      ov_wheel_input: ${{ steps.openvino_download.outputs.ov_wheel_input }}
+      ov_wheel_source: ${{ steps.openvino_download.outputs.ov_wheel_source }}
     timeout-minutes: 10
     defaults:
       run:
         shell: bash
-    runs-on: ubuntu-20.04
+    runs-on: aks-linux-2-cores-8gb
+    container:
+      image: 'openvinogithubactions.azurecr.io/openvino_provider:0.1.0'
+      volumes: [ "/mount:/mount" ]
+    continue-on-error: true
 
     steps:
-      - name: Download OpenVINO build
-        id: openvino_download
-        run: |
-          wget ${{ env.OV_TARBALL}} --progress=bar:force:noscroll -O openvino_package.tar.gz
-          tar -tvf openvino_package.tar.gz
-        continue-on-error: true
-
-      #
-      # Upload to artifacts
-      #
-
-      - name: Upload openvino package
-        if: steps.openvino_download.outcome == 'success'
-        uses: actions/upload-artifact@v4
-        with:
-          name: openvino_package
-          path: openvino_package.tar.gz
-          if-no-files-found: 'error'
+    - uses: openvinotoolkit/openvino/.github/actions/openvino_provider@master
```

Comment:
is it OK to use master?
I think on master we can rename some args, etc which is not BW compatible.

Reply:
I thought about this, ideally we should download an action from openvino_provider@${{ env.OV_BRANCH }}, but "uses" key doesn't allow passing variables: https://github.com/orgs/community/discussions/25246

We can use a specific hash instead and update it manually when needed, or call OV repo clone from env.OV_BRANCH as a separate step beforehand

Reply:
I think we can update commit manually, because I hope OV provider will be stable rather sooner than later and we can stick to stable (old commit) version.

Reply:
TODO: use particular commit when PR is ready for merge